src/site/content/en/blog/fugu-status/index.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: New capabilities status
-subhead: Web apps should be able to do anything iOS/Android/desktop apps can. Google wants to make it possible for you to build and deliver apps on the open web that have never been possible before.
+subhead: Web apps should be able to do anything iOS/Android/desktop apps can. We want to make it possible for you to build and deliver apps on the open web that have never been possible before.
```

Comment:
We can't say 'we'.

Reply:
@jpmedley PTAL, I have no rephrased to say the project members.